.github/workflows/ci.yml

```diff
@@ -75,6 +75,16 @@ jobs:
           distribution: temurin
           java-version: "11"
 
+      - name: Checkout ocaml env
```

Comment:
How about add a `Setup OCaml toolchain` like rust under `.github/actions/setup_ocaml`?

Reply:
good idea, i'll try it.